### DISCUSSION TITLE
How is the documentation for this repo created?

### DISCUSSION THREAD

**Discussion by sugatoray** [zou-group/textgrad#24] [Q&A] [answered]
The API documentation is maintained here: https://textgrad.readthedocs.io/en/latest/index.html.

However, I could not find any CI or other method within the repo itself that produces the rendered docs-site. Could you please shed some light on this aspect?

- What is the tech-stack used for creating the docs page? (Sphinx/MkDocs/Jupytext/other)
- Where is it maintained?

Thank you.

cc: @vinid @mertyg

**Comment by vinid** [comment, marked as answer]
There's a branch docs in the repo that has a webhook connected to the readthedocs website. Changes to ti that branch arw automatically propagates to the website

**Reply by sugatoray** [reply]
Thank you.
